extensions/teams/package.json

```diff
@@ -24,15 +24,15 @@
     },
     "dependencies": {
         "botbuilder": "^4.22.1",
-        "restify": "^11.1.0"
+        "restify": "^10.0.0"
     },
     "devDependencies": {
-        "@types/node": "^18.19.26",
-        "@types/restify": "^8.5.12",
+        "@types/node": "^16.0.0",
+        "@types/restify": "^8.5.5",
         "env-cmd": "^10.1.0",
-        "nodemon": "^3.1.0",
+        "nodemon": "^2.0.7",
         "shx": "^0.3.3",
-        "ts-node": "^10.9.2",
-        "typescript": "^5.4.3"
+        "ts-node": "^10.4.0",
+        "typescript": "^4.4.4"
```

Comment:
It looks like all of these dependencies have been downgraded, was that intentional?

Reply:
Yes. We are running the library versions based on the Teams Toolkit recommendations.

Reply:
> Yes. We are running the library versions based on the Teams Toolkit recommendations.

I don't suppose you could point me to those recommendations? I looked here https://github.com/Microsoft/botbuilder-js but couldn't find them. Thanks!

Reply:
The Teams Toolkit provides you with a base template for several different scenarios: bot, tab, messaging extension, etc. Those base templates contain the latest supported libraries for a given scenario. I've attempted to update some of the libraries to their latest version with mixed results. Typically, I stay with the base template until an update to the Teams Toolkit is released.

Reply:
Ok gotcha, do you think it's worth turning off dependabot for the teams extension? Otherwise it will just keep trying to bump the versions. If so, could you disable that as part of this PR? Dependabot will immediately try and re-bump the versions otherwise 😅

Reply:
Understood. Just checked in the update.